.markdownlint-cli2.jsonc

```diff
@@ -0,0 +1,14 @@
+// https://github.com/DavidAnson/markdownlint-cli2#markdownlint-cli2jsonc
+{
+  "config": {
+    "MD013": false,
+    "MD024": false,
```

Comment:
This rules is: "[MD024](https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md024.md) no-duplicate-heading - Multiple headings with the same content"

Do you think we should keep it? or is it out of scope for this PR?

Reply:
I'd say this is out of scope. All of these rules (except the MD004/ul-style rule) are what we already had in place in `.markdownlint.json`:

```
{
  "MD013": false,
  "MD024": false,
  "MD033": false,
  "MD041": false,
  "MD026": false
}
```

FWIW, when I try to remove this rule I get 349 lint errors. Most are in the CHANGELOG files which, as expected, duplicate headers for each release (`### :boom: Breaking Change`, etc.). There were a couple dupes in the README which I don't think are a problem.